hw2/Index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>hw2</title>
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/fonts.css">
+	<link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+	<div class="wrapper">
+		<header>
+			<img src="img/logo.png">
+			<ul class="top-menu">
+				<li><a class="top-menu_item" href="#">Главная</a></li>
```

Comment:
moreover better call li - item, a - link like below. Anyway do the same way everywhere